plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -18,7 +18,8 @@ private MLCommonsSettings() {}
         .intSetting("plugins.ml_commons.max_model_on_node", 10, 0, 1000, Setting.Property.NodeScope, Setting.Property.Dynamic);
     public static final Setting<Integer> ML_COMMONS_MAX_LOAD_MODEL_TASKS_PER_NODE = Setting
         .intSetting("plugins.ml_commons.max_load_model_tasks_per_node", 10, 0, 10, Setting.Property.NodeScope, Setting.Property.Dynamic);
-
+    public static final Setting<Integer> ML_COMMONS_MAX_ML_TASK_PER_NODE = Setting
+        .intSetting("plugins.ml_commons.max_ml_task_per_node", 10, 0, 10, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
The upper limit seems small, change to 10k?
```suggestion
        .intSetting("plugins.ml_commons.max_ml_task_per_node", 10, 0, 10000, Setting.Property.NodeScope, Setting.Property.Dynamic);
```